@AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_wcaBtnHandleControl.sqf

```diff
@@ -20,13 +23,13 @@ switch (_control) do {
         playsound "fza_ah64_switch_flip4";
     };
     case "mstrCaut": {
-        _heli setVariable ["fza_ah64_mstrCautLightOn", !(_heli getVariable "fza_ah64_mstrCautLightOn"), true];
+        _heli setVariable ["fza_ah64_mstrCautLightOn", false, true];
```

Comment:
why did you change this if you didn't change it everywhere else? 

we either need to use use:

!(_heli getVariable "fza_ah64_emerHydOn")

or explicitly set everything to false as you've done here

Reply:
can you toggle master warning/caution light on/off, because thats whats happeniing

Reply:
Nope, your change here is good, didn't catch onto the issue at first.